compiler/parser.jou

```diff
@@ -1138,6 +1259,17 @@ class Parser:
             if t->kind == TokenKind.Decorator:
                 decorate(&result, t->short_string, t->location)
 
+        # Inside a class, variable declarations become fields.
```

Comment:
Instead of this, it would be better to set it directly to the correct type, similar to FunctionDef/MethodDef